testing/examples/author_tests.rs

```diff
@@ -67,50 +67,52 @@ async fn main() {
 
 	thread::sleep(Duration::from_secs(6)); // Wait a little to avoid transaction too low priority error.
 	let xt2 = api.balance_transfer_allow_death(bob.clone(), 1000);
-	let report = api.submit_and_watch_extrinsic_until(xt2, XtStatus::Ready).unwrap();
+	let report = api
+		.submit_and_watch_extrinsic_until_without_events(xt2, XtStatus::Ready)
+		.unwrap();
 	assert!(report.block_hash.is_none());
-	println!("Success: submit_and_watch_extrinsic_until Ready");
+	println!("Success: submit_and_watch_extrinsic_until_without_events Ready");
```

Comment:
I'd leave the function testing as is. `submit_and_watch_extrinsic_until_without_events ` is automatically tested internally when calling `submit_and_watch_extrinsic_until`. So I'd rather test the wrapper function instead of the internal one.

Reply:
Though, implement one or two calls, just as a showcase for us and others on how to use the function.

Reply:
?
We have in this test:
let events = api.submit_and_watch_extrinsic_until(xt6, XtStatus::InBlock)

Reply:
The idea of this crate was to test every possible outcome of each function in the api-client, that was not unit testable (due to Substrate node dependency). 

For example, one case that is currently not tested, is if ` api.submit_and_watch_extrinsic_until` really works for statuses below `InBlock` E.g. does this part of the code
```rust
if watch_until < XtStatus::InBlock {
	return Ok(report)
}
```
really work? Is the report returned without events? Or does it error upon not finding any associated events onchain?